app/assets/config/manifest.js

```diff
@@ -0,0 +1,6 @@
+//= link admin.css
+//= link dues.js
+//= link membership_note.js
+//
+//= link application.css
+//= link application.js
```

Comment:
newline is missing (surprised rubocop didn't complain about this)

Reply:
I guess Rubocop does not check JS files :wink: Fixed.